test/cypress/integration/settings/mask-moto-card-number-and-security-code.cy.test.js

```diff
@@ -0,0 +1,167 @@
+'use strict'
+
+const userStubs = require('../../stubs/user-stubs')
+const gatewayAccountStubs = require('../../stubs/gateway-account-stubs')
+
+describe('MOTO mask security section', () => {
+  const userExternalId = 'cd0fa54cf3b7408a80ae2f1b93e7c16e'
+  const gatewayAccountId = 42
+  const serviceName = 'Purchase a positron projection permit'
+
+  function setupMotoStubs (opts = {}) {
+    let stubs = []
+    let user
+
+    if (opts.readonly) {
+      const role = {
+        permissions: [
+          {
+            name: 'transactions-details:read',
+            description: 'ViewTransactionsOnly'
+          },
+          {
+            name: 'moto-mask-input:read',
+            description: 'ViewMaskMotoOnly'
+          }
+        ]
+      }
+      user = userStubs.getUserSuccess({ userExternalId, gatewayAccountId, serviceName, role })
+    } else {
+      user = userStubs.getUserSuccess({ userExternalId, gatewayAccountId, serviceName })
+    }
+
+    const gatewayAccount = gatewayAccountStubs.getGatewayAccountSuccess({ gatewayAccountId, paymentProvider: opts.gateway, allowMoto: opts.allowMoto, motoMaskCardNumber: opts.motoMaskCardNumber, motoMaskSecurityCode: opts.motoMaskSecurityCode })
+    const card = gatewayAccountStubs.getAcceptedCardTypesSuccess({ gatewayAccountId, updated: false, maestro: opts.maestro })
+    const patchUpdateMotoMaskCardNumber = gatewayAccountStubs.patchUpdateMotoMaskCardNumber({ motoMaskCardNumber: opts.motoMaskCardNumber })
+    const patchUpdateMotoMaskSecurityCode = gatewayAccountStubs.patchUpdateMotoMaskSecurityCode({ motoMaskCardSecurityCode: opts.motoMaskCardSecurityCode })
+
+    stubs.push(user, gatewayAccount, card, patchUpdateMotoMaskCardNumber, patchUpdateMotoMaskSecurityCode)
+
+    cy.task('setupStubs', stubs)
+  }
+
+  beforeEach(() => {
+    Cypress.Cookies.preserveOnce('session', 'gateway_account')
+  })
+
+  describe('When accessing MOTO mask settings', () => {
+    describe('when gateway account has allowMoto=false', () => {
+      beforeEach(() => {
+        setupMotoStubs({ allowMoto: false })
+      })
+
+      it('should not show mask security section', () => {
+        cy.setEncryptedCookies(userExternalId, gatewayAccountId)
+        cy.visit('/settings')
+        cy.get('#moto-mask-security-settings-heading').should('not.exist')
+      })
+    })
+
+    describe('mask card number - when user has read permission and card number mask disabled', () => {
+      beforeEach(() => {
+        setupMotoStubs({ readonly: true, allowMoto: true, motoMaskCardNumber: false })
+      })
+
+      it('should show radios as disabled and card number mask disabled', () => {
+        cy.visit('/settings')
+        cy.get('.govuk-summary-list__key').eq(4).should('contain', 'Hide card number')
```

Comment:
The [figma](https://www.figma.com/file/egLWv6GVXnhcDtU2ToxCnX/CVV-number-masking?node-id=0%3A1) diagram shows the options "Mask CVV number", "Mask card number" and "Mask CVV and card numbers". Why is it "Hide card number" here?